Composer/packages/client/src/constants.tsx

```diff
@@ -525,3 +525,11 @@ export const defaultTeamsManifest: TeamsManifest = {
 
 export const defaultBotPort = 3979;
 export const defaultBotEndpoint = `http://localhost:${defaultBotPort}/api/messages`;
+
+const DAYS = 1000 * 60 * 60 * 24; // ms per day
```

Comment:
```suggestion
const MS_PER_DAY = 1000 * 60 * 60 * 24; 
```

Reply:
I thought about that, but I like the way it makes "90 * DAYS" legible a few lines farther down.